tests/illumina_test.py

```diff
@@ -42,7 +42,7 @@ def test_parse_header(self, sep: str, sano: bool) -> None:
     @pytest.mark.parametrize("sano", (False, True))
     def test_generate_blocks(self, sano: bool) -> None:
```

Comment:
We can use this to test both sorted and unsorted inputs in one go, since the test code itself is the same - something like:

```python
    @pytest.mark.parametrize("sano,sort_", itertools.product((False, True),(True, False)))
    def test_generate_blocks(self, sano: bool, sorted_:bool) -> None:
```

See https://docs.python.org/3/library/itertools.html#itertools.product